### ISSUE TITLE
Make "Study" suffix for display labels of data spaces optional or customizable

### ISSUE THREAD

**Issue by peterbanda** [ada-discovery/ada-issues#56]

**Comment by peterbanda** [comment]
Alternatively we can drop "study" suffix altogether and let Ada's admins decide which data space is study and which not by adjusting their names.

**Comment by sherzinger** [comment]
Agreed, we can drop the study suffix I think, especially since it does not always hold true. If necessary we incorporate it into the name where applicable. I can take care of this issue.
